From 1bbc36ee80778819c872a60b6a6a3605bf217524 Mon Sep 17 00:00:00 2001
From: Pierre Lucas <plucas@esoftthings.com>
Date: Wed, 6 Nov 2019 15:31:03 +0100
Subject: [PATCH 1/1] cert_header_sa3: update for eView

Update informations relative to R7 paylaod (here eView) which are
stored in cert_header_sa3.sre):
   - increased the size of eView payload.
   - set eView start address to 0x40040000 (DDR0).

Signed-off-by: Pierre Lucas <plucas@esoftthings.com>
---
 loader/rcar_bl2_addr.c       | 2 +-
 tools/dummy_create/V3H/sa3.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/loader/rcar_bl2_addr.c b/loader/rcar_bl2_addr.c
index 2228257..dca504d 100644
--- a/loader/rcar_bl2_addr.c
+++ b/loader/rcar_bl2_addr.c
@@ -42,7 +42,7 @@
 /*****************************************************************************
  *	Definitions
  *****************************************************************************/
-#define RTOS_MAX_SIZE			(1024U * 1024U)
+#define RTOS_MAX_SIZE			(4U * 1024U * 1024U)
 #define MAIN_LOADER_MAX_SIZE		( 178U * 1024U)
 
 #define FLASH_BASE			(0x08000000U)
diff --git a/tools/dummy_create/V3H/sa3.c b/tools/dummy_create/V3H/sa3.c
index 704cf96..822133b 100644
--- a/tools/dummy_create/V3H/sa3.c
+++ b/tools/dummy_create/V3H/sa3.c
@@ -31,9 +31,9 @@
 /* SA3 */
 /* 0x000C0154 */
 //const unsigned int __attribute__ ((section (".sa3_rtos_addr"))) rtos_addr = 0x480A0800;
-const unsigned int __attribute__ ((section (".sa3_rots_addr"))) rtos_addr = 0x480A0800;
+const unsigned int __attribute__ ((section (".sa3_rots_addr"))) rtos_addr = 0x40040000;
 /* 0x000C0264 */
-const unsigned int __attribute__ ((section (".sa3_rtos_size"))) rtos_size = 0x00008000;
+const unsigned int __attribute__ ((section (".sa3_rtos_size"))) rtos_size = 0x00100000;
 /* 0x000C0554 */
 const unsigned int __attribute__ ((section (".sa3_bl2loader_addr"))) bl2loader_addr = 0xeb244000;
 /* 0x000C0664 */
-- 
2.17.1

